docs/pages/setup/admin/daemon.mdx

```diff
@@ -143,15 +163,44 @@ To upgrade a host to a newer version of Teleport, you must:
 
 </Admonition>
 
-## Further reading
+
+## Understanding Teleport Daemon Command Line Options for Unit File Configuration
```

Comment:
Should this content be in the [CLI reference](https://github.com/gravitational/teleport/blob/d6e3202a49627d3e3043c5ebd7545e4fc5b64da3/docs/pages/setup/reference/cli.mdx)? Then we can add a link to this section of the CLI reference in the "Next steps" section.

Reply:
It should be, but for now the CLI reference, specifically for the `teleport` command appears to be out of date beyond just the daemon config, and I felt it may cause friction if we link to a patchwork fix for the CLI reference to address systemd without a more focused effort for `teleport` subcommands as a whole. I created a separate issue to address this and it's next on my list of things to do should no other task need to be prioritized first. 

For the time being I felt it would be best to update the doc highlighting the feature directly until the CLI reference gets a larger overhaul since other features of the CLI users may need aren't documented, but I am happy to begin that work now within this PR if you feel that may be better. 

https://github.com/gravitational/teleport/issues/15592